src/travel-details-map-screen/TravelDetailsMapScreenComponent.tsx

```diff
@@ -238,29 +250,64 @@ const LiveVehicle = ({
     };
   })();
 
-  if (!vehicle.location || zoomLevel < FOLLOW_MIN_ZOOM_LEVEL) return null;
+  if (
+    !vehicle.location ||
+    zoomLevel < FOLLOW_MIN_ZOOM_LEVEL ||
+    vehicle.bearing === undefined
```

Comment:
I was wondering if in the case that bearing (angle) is not provided, then we should show the bus only.